bootstrap-languages/agent-language/hc-dna/workdir/happ.yaml

```
@@ -0,0 +1,11 @@
+manifest_version: '1'
+name: ad4m-agent-language
+description: Core bootstrap network for ad4m storing AgentExpressions per DID
+roles:
+- name: agent-store
+  provisioning:
+    strategy: create
+    deferred: false
+  dna:
+    bundled: ./agent-store.dna
+    
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)